Src/ElasticScale.Client/ShardManagement/SqlStore/SqlShardMapManagerCredentials.cs

```diff
@@ -135,24 +173,51 @@ internal static void EnsureCredentials(SqlConnectionStringBuilder connectionStri
                 return;
             }
 
-            // UserID must be set when integrated authentication is disabled.
-            if (string.IsNullOrEmpty(connectionString.UserID))
+            // If secure credential not specified, verify that user/pwd are in the connection string. If secure credential
```

Comment:
I assume SqlCredential validates that username & password are not null. Is that right?

Reply:
Correct, if null is provided for either username/password to SqlCredential, it throws: "Unhandled Exception: System.ArgumentNullException: Value cannot be null."